### PR TITLE
chore: remove unnecessary code

### PR DESCRIPTION
**Pull request by ctrlc03**
Remove unnecessary code

### CODE REVIEW

File: src/pages/api/og.tsx
Comment (kittybest) [inline src/pages/api/og.tsx:1]: this file cannot be removed, it's used inside config file (metadata).

File: src/utils/csv.ts
Comment (kittybest) [inline src/utils/csv.ts:1]: this would be used when we need the function of import / export csv file of ballot

**Review by kittybest — COMMENTED** [review]
thanks! just left some comments

File: src/pages/api/og.tsx
Comment (ctrlc03) [reply src/pages/api/og.tsx:1]: 😨

File: src/utils/csv.ts
Comment (ctrlc03) [reply src/utils/csv.ts:1]: not sure we wanna have that feature in the future? one less way for users to show/lose/disclose their plaintext ballot?

Comment (kittybest) [reply src/utils/csv.ts:1]: then we could remove it now~

**Review by kittybest — APPROVED** [review]
thanks!

**Pull request merged**